Zend/zend_builtin_functions.c

```diff
@@ -1420,30 +1415,22 @@ ZEND_FUNCTION(get_defined_functions)
 	zval internal, user;
 	zend_string *key;
 	zend_function *func;
-	zend_bool exclude_disabled = 0;
-	char *disable_functions = NULL;
+	zend_bool exclude_disabled = 1;
 
 	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|b", &exclude_disabled) == FAILURE) {
 		RETURN_THROWS();
 	}
 
+	/* exclude_disabled = 0 is ignored. */
```

Comment:
Should we deprecate that parameter right away?

Reply:
Yeah, I think we can deprecated passing false to this parameter right away, and silently accept true.

Should we already deprecate ReflectionFunction::isDisabled()?

Reply:
IMHO, yes.

Reply:
I've added both deprecations.